### PR TITLE
Add inorganic compounds batch 2

### PR DESCRIPTION
**Pull request by sumitnegi7**

### CODE REVIEW

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 1]
```diff
@@ -509,4 +509,410 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+    meltingOutcomes: [],
+    boilingOutcomes: [],
+    solubilitySheet: {
+      coldWater: {
+        nature: SolubilityNature.Insoluble,
+      },
+    },
+  },
+
+  OrthoClaseAluminumPottasiumSilicate: {
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:534]:
```suggestion
  OrthoclaseAluminumPottasiumSilicate: {
```

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 2]
```diff
@@ -509,4 +509,410 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+      },
+    },
+  },
+
+  AluminumPottasiumTartrate: {
+    name: 'Aluminum potassium tartrate',
+    formula: 'AlK(C4H4O6)2',
+    formulaWeight: 362.22,
+    color: { nature: Colors.Colorless },
+    // Does not have crystalline form in the table
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:561]:
```suggestion
```

Noted the comment. Can be removed now.

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 3]
```diff
@@ -509,4 +509,410 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+
+  AluminumSulfate: {
+    name: 'Aluminum sulfate',
+    formula: 'Al2(SO4)3',
+    formulaWeight: 342.15,
+    color: { nature: Colors.White },
+    crystallineForms: {
+      nature: CrystallineForms.Crystalline,
+    },
+    refractiveIndex: { value: 1.529 },
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:637]:
```suggestion
```

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 4]
```diff
@@ -509,4 +509,410 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+          value: 100,
+          units: 'C',
+        },
+      },
+    },
+  },
+
+  Alum: {
+    name: 'Alum, ammonium (tschermigite)',
+    formula: 'Al2(SO4)3.(NH4)2SO4.',
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:666]:
```suggestion
    formula: 'Al2(SO4)3.(NH4)2SO4.24H20',
```

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 5]
```diff
@@ -509,4 +509,410 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+        nature: SolubilityNature.Soluble,
+        value: 3.9,
+        temperature: {
+          value: 0,
+          units: 'C',
+        },
+      },
+      hotWater: {
+        nature: SolubilityNature.Soluble,
+        value: Infinity,
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:698]:
```suggestion
        value: Number.Infinity,
```

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 6]
```diff
@@ -509,4 +509,410 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+        temperature: {
+          value: 100,
+          units: 'C',
+        },
+      },
+      acid: {
+        nature: SolubilityNature.Decomposes,
+      },
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:706]: Looks wrong

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 7]
```diff
@@ -509,4 +509,410 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+    },
+  },
+
+  AmmoniumChrome: {
+    name: 'ammonium chrome',
+    formula: 'Cr2(SO4)3.(NH4)2SO4.24H2O',
+    formulaWeight: 956.69,
+    color: [
+      {
+        nature: Colors.Green, // Color is given Green or Violet Please see if you like this structure or suggest something else
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:716]:
```suggestion
        nature: Colors.Green,
```

Looks good

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 8]
```diff
@@ -509,4 +509,409 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+        },
+      },
+      acid: {
+        nature: SolubilityNature.Decomposes,
+      },
+    },
+  },
+
+  AmmoniumChrome: {
+    name: 'ammonium chrome',
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:710]:
```suggestion
    name: 'Ammonium chrome',
```

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 9]
```diff
@@ -509,4 +509,409 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+        gas: false,
+      },
+      ethylAlcohol95percent: {
+        nature: SolubilityNature.Soluble,
+      },
+    },
+  },
+
+  AmmoniumIron: {
+    name: 'ammonium iron',
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:748]:
```suggestion
    name: 'Ammonium iron',
```

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 10]
```diff
@@ -509,4 +509,409 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+        gas: false,
+      },
+      ethylAlcohol95percent: {
+        nature: SolubilityNature.Insoluble,
+      },
+    },
+  },
+
+  AmmoniumPotassium: {
+    name: 'ammonium potassium (kalnite)',
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:781]:
```suggestion
    name: 'Ammonium potassium (kalnite)',
```

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 11]
```diff
@@ -509,4 +509,409 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+        temperature: {
+          value: 93,
+          units: 'C',
+        },
+        gas: false,
+      },
+    },
+  },
+
+  AmmoniumPotassiumChrome: {
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:827]:
```suggestion
  PotassiumChrome: {
```

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 12]
```diff
@@ -509,4 +509,409 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+    name: 'ammonium potassium chrome',
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:828]:
```suggestion
    name: 'Potassium chrome',
```

File: src/physical-and-chemical-data/data/inorganic-compounds.ts [hunk 13]
```diff
@@ -509,4 +509,409 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+        gas: false,
+      },
+      ethylAlcohol95percent: {
+        nature: SolubilityNature.Insoluble,
+      },
+    },
+  },
+
+  AmmoniumSodium: {
+    name: 'ammonium sodium',
```
Comment (maneetgoyal) [inline src/physical-and-chemical-data/data/inorganic-compounds.ts:867]:
```suggestion
    name: 'Ammonium sodium',
```

**Review by maneetgoyal — COMMENTED** [review]
- Reviewed all.
- Bunch of small issues are there.
- Take a look more time to ensure no errors.

File: src/physical-and-chemical-data/data/inorganic-compounds.ts (hunk 5 above)
Comment (sumitnegi7) [reply src/physical-and-chemical-data/data/inorganic-compounds.ts:696]: Number.Infinity is throwing error `any
Property 'Infinity' does not exist on type 'NumberConstructor'`

**Comment by sumitnegi7** [comment]
Sir @maneetgoyal  rechecked it one mistake was there pushed a commit for it https://github.com/manufac-analytics/perry/pull/6/commits/6ef6a413103de3b655f232b8fc16939549cefc8d Rest LGTM

File: src/physical-and-chemical-data/data/inorganic-compounds.ts (hunk 5 above)
Comment (maneetgoyal) [reply src/physical-and-chemical-data/data/inorganic-compounds.ts:695]: Current is okay then

**Review by maneetgoyal — APPROVED** [review]

**Pull request merged**